src/lib/edge-functions/proxy.js

```diff
@@ -91,8 +92,9 @@ const initializeProxy = async ({
 
     if (!registry) return
 
-    // Setting header with geolocation.
+    // Setting header with geolocation and site info.
     req.headers[headers.Geo] = JSON.stringify(geoLocation)
+    req.headers[headers.Site] = btoa(JSON.stringify({ id: siteInfo.id, name: siteInfo.name, url: siteInfo.url }))
```

Comment:
Where is `btoa` coming from? It is not imported or anything and I do not think it is a global.

Reply:
I see https://developer.mozilla.org/en-US/docs/Web/API/btoa, but it is only available as of node 16. So we cannot use it.

Reply:
Hopefully that is the problem! Thank you so. much. 

Somehow because of the use of `atob()` on the receiving end I thought it was ok but I should have looked at the version support.

Will be diving into it!

Reply:
👍 that was it!

Reply:
Ah, good catch!